[2025-02-05T00:28:19.433+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T00:28:19.443+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.localiza_dw_test manual__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-05T00:28:19.448+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.localiza_dw_test manual__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-05T00:28:19.449+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T00:28:19.457+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): model_credit_fraud.localiza_dw_test> on 2025-02-05 00:00:00+00:00
[2025-02-05T00:28:19.461+0000] {standard_task_runner.py:63} INFO - Started process 1723 to run task
[2025-02-05T00:28:19.463+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_credit_fraud', 'model_credit_fraud.localiza_dw_test', 'manual__2025-02-05T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_fraud_credit.py', '--cfg-path', '/tmp/tmpq69l7p9j']
[2025-02-05T00:28:19.464+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask model_credit_fraud.localiza_dw_test
[2025-02-05T00:28:19.497+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.localiza_dw_test manual__2025-02-05T00:00:00+00:00 [running]> on host eb20c43aeeb4
[2025-02-05T00:28:19.559+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_credit_fraud' AIRFLOW_CTX_TASK_ID='model_credit_fraud.localiza_dw_test' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T00:00:00+00:00'
[2025-02-05T00:28:19.562+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T00:28:21.337+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T00:28:21.338+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpf7doa9qy from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T00:28:21.349+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_credit_fraud__model_credit_fraud/target/partial_parse.msgpack
[2025-02-05T00:28:21.353+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T00:28:21.358+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T00:28:21.359+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211/profiles.yml.
[2025-02-05T00:28:21.360+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['test', '--select', 'path:models/marts/credit_fraud path:models/staging/credit_fraud path:snapshots/credit_fraud', '--project-dir', '/tmp/tmpf7doa9qy', '--profiles-dir', '/tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211', '--profile', '***', '--target', 'prod']
 in /tmp/tmpf7doa9qy
[2025-02-05T00:28:21.369+0000] {logging_mixin.py:188} INFO - 00:28:21  Running with dbt=1.6.1
[2025-02-05T00:28:21.443+0000] {logging_mixin.py:188} INFO - 00:28:21  Registered adapter: postgres=1.6.1
[2025-02-05T00:28:21.603+0000] {logging_mixin.py:188} INFO - 00:28:21  Found 3 models, 1 snapshot, 5 tests, 1 source, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T00:28:21.612+0000] {logging_mixin.py:188} INFO - 00:28:21
[2025-02-05T00:28:21.660+0000] {logging_mixin.py:188} INFO - 00:28:21  Concurrency: 1 threads (target='prod')
[2025-02-05T00:28:21.662+0000] {logging_mixin.py:188} INFO - 00:28:21
[2025-02-05T00:28:21.665+0000] {logging_mixin.py:188} INFO - 00:28:21  1 of 5 START test locations .................................................... [RUN]
[2025-02-05T00:28:21.710+0000] {logging_mixin.py:188} INFO - 00:28:21  1 of 5 PASS locations .......................................................... [PASS in 0.04s]
[2025-02-05T00:28:21.712+0000] {logging_mixin.py:188} INFO - 00:28:21  2 of 5 START test locations .................................................... [RUN]
[2025-02-05T00:28:22.537+0000] {logging_mixin.py:188} INFO - 00:28:22  2 of 5 WARN 1 locations ........................................................ [WARN 1 in 0.82s]
[2025-02-05T00:28:22.540+0000] {logging_mixin.py:188} INFO - 00:28:22  3 of 5 START test not_null_stg_credit_fraud_amount ............................. [RUN]
[2025-02-05T00:28:24.628+0000] {logging_mixin.py:188} INFO - 00:28:24  3 of 5 WARN 50000 not_null_stg_credit_fraud_amount ............................. [WARN 50000 in 2.09s]
[2025-02-05T00:28:24.631+0000] {logging_mixin.py:188} INFO - 00:28:24  4 of 5 START test not_null_stg_credit_fraud_risk_score ......................... [RUN]
[2025-02-05T00:28:26.868+0000] {logging_mixin.py:188} INFO - 00:28:26  4 of 5 WARN 50226 not_null_stg_credit_fraud_risk_score ......................... [WARN 50226 in 2.24s]
[2025-02-05T00:28:26.870+0000] {logging_mixin.py:188} INFO - 00:28:26  5 of 5 START test not_null_stg_credit_fraud_timestamp .......................... [RUN]
[2025-02-05T00:28:27.793+0000] {logging_mixin.py:188} INFO - 00:28:27  5 of 5 PASS not_null_stg_credit_fraud_timestamp ................................ [PASS in 0.92s]
[2025-02-05T00:28:27.837+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.846+0000] {logging_mixin.py:188} INFO - 00:28:27  Finished running 5 tests in 0 hours 0 minutes and 6.22 seconds (6.22s).
[2025-02-05T00:28:27.859+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.861+0000] {logging_mixin.py:188} INFO - 00:28:27  Completed with 3 warnings:
[2025-02-05T00:28:27.863+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.864+0000] {logging_mixin.py:188} INFO - 00:28:27  Warning in test locations (models/staging/credit_fraud/schema.yml)
[2025-02-05T00:28:27.865+0000] {logging_mixin.py:188} INFO - 00:28:27  Got 1 result, configured to warn if != 0
[2025-02-05T00:28:27.866+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.867+0000] {logging_mixin.py:188} INFO - 00:28:27    compiled Code at target/compiled/***/models/staging/credit_fraud/schema.yml/locations.sql
[2025-02-05T00:28:27.868+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.869+0000] {logging_mixin.py:188} INFO - 00:28:27  Warning in test not_null_stg_credit_fraud_amount (models/staging/credit_fraud/schema.yml)
[2025-02-05T00:28:27.870+0000] {logging_mixin.py:188} INFO - 00:28:27  Got 50000 results, configured to warn if != 0
[2025-02-05T00:28:27.871+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.872+0000] {logging_mixin.py:188} INFO - 00:28:27    compiled Code at target/compiled/***/models/staging/credit_fraud/schema.yml/not_null_stg_credit_fraud_amount.sql
[2025-02-05T00:28:27.874+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.876+0000] {logging_mixin.py:188} INFO - 00:28:27  Warning in test not_null_stg_credit_fraud_risk_score (models/staging/credit_fraud/schema.yml)
[2025-02-05T00:28:27.877+0000] {logging_mixin.py:188} INFO - 00:28:27  Got 50226 results, configured to warn if != 0
[2025-02-05T00:28:27.878+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.879+0000] {logging_mixin.py:188} INFO - 00:28:27    compiled Code at target/compiled/***/models/staging/credit_fraud/schema.yml/not_null_stg_credit_fraud_risk_score.sql
[2025-02-05T00:28:27.880+0000] {logging_mixin.py:188} INFO - 00:28:27
[2025-02-05T00:28:27.881+0000] {logging_mixin.py:188} INFO - 00:28:27  Done. PASS=2 WARN=3 ERROR=0 SKIP=0 TOTAL=5
[2025-02-05T00:28:28.613+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T00:28:28.640+0000] {local.py:494} INFO - Inlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.agg_locations_risk_score', extra=None), Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.stg_credit_fraud', extra=None)]
[2025-02-05T00:28:28.641+0000] {local.py:495} INFO - Outlets: []
[2025-02-05T00:28:28.642+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T00:28:28.643+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T00:28:28.652+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_credit_fraud to None, run_after=None
[2025-02-05T00:28:28.683+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T00:28:28.699+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_credit_fraud, task_id=model_credit_fraud.***_test, run_id=manual__2025-02-05T00:00:00+00:00, execution_date=20250205T000000, start_date=20250205T002819, end_date=20250205T002828
[2025-02-05T00:28:28.739+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T00:28:28.750+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T00:28:28.757+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
